templates/taxi/driver_form.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update driver license, Create driver" }}</h1>
```

Comment:
The `yesno` filter is used with `object` to determine the heading. Ensure that `object` is properly defined in the context. Typically, `object` is used in class-based views, so verify that the view rendering this template provides `object` in the context. If using a function-based view, you might need to adjust the context variable accordingly.